eng/Version.Details.xml

```diff
@@ -16,6 +16,10 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>7b2cd1ee7169143248a7da9fd749caf22affa624</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.TemplateEngine.Tasks" Version="7.0.100-rc.1.22409.14">
+      <Uri>https://github.com/dotnet/templating</Uri>
+      <Sha>efd5e59cbe2751b14f5331d122ccba27dd788cf4</Sha>
+    </Dependency>
```

Comment:
Should this go under `ToolsetDependencies` or `ProductDependencies`? I think it mainly just shows something different on Maestro's website depending on what you pick.

Reply:
I'm not sure, [maui](https://github.com/dotnet/maui/blob/8e8b2973c7ca42082957e9b60d9407c85d158608/eng/Version.Details.xml#L70) uses product but [macios](https://github.com/xamarin/xamarin-macios/blob/4221018abada0941f34a787fc4d40561e296db3e/eng/Version.Details.xml#L30) uses toolset. I think toolset probably makes more sense for how it is used so I will move it.